WordPressAuthenticator/Signin/LoginViewController.swift

```diff
@@ -425,10 +425,19 @@ extension LoginViewController {
     /// Updates the LoginFields structure, with the specified Google User + Token + Email.
     ///
     func updateLoginFields(googleUser: GIDGoogleUser, googleToken: String, googleEmail: String) {
-        loginFields.emailAddress = googleEmail
-        loginFields.username = googleEmail
+        updateLoginFields(
+            email: googleEmail,
+            username: googleEmail,
+            fullName: googleUser.profile?.name ?? "",
+            googleToken: googleToken
+        )
+    }
+
+    func updateLoginFields(email: String, username: String, fullName: String, googleToken: String) {
```

Comment:
This `username` argument is not used, should we get rid of it? Also, any reason adding another `updateLoginFields` function here, rather than do the assignments in the one above?

Reply:
What I should have done with that `username` parameter was using it in the method and then passing `.email` at callsite. I guess I was trying to cover my basis.

But your observation made me realize the version with the `GIDGoogleUser` input is unused, and this new one is only used by it. So, I removed both of them.

Reply:
Yay, more deletions!